Kitodo/src/main/java/de/sub/goobi/forms/HelperForm.java

```diff
@@ -304,4 +305,13 @@ public String getUserAgent() {
     public boolean getUserStepDoneSearchEnabled() {
         return ConfigCore.getBooleanParameter("withUserStepDoneSearch");
     }
+    public static void saveMessage() {
```

Comment:
1) Public method without JavaDoc - please add JavaDoc.
2) Why you are adding a new method to display waring / error messages? There are a lot of them already existing and can be used.